lib/bank/account.rb

```diff
@@ -0,0 +1,37 @@
+module Bank
+  class Account
+    # attr_accessor creates an implicit balance method
+    attr_accessor :id, :balance
```

Comment:
As attr_accessors, id and balance are both read and writeable. In the context of a bank account, is that something you would want a user to be able to alter directly?